frontend/src/js/components/FinalConfirmation.tsx

```diff
@@ -0,0 +1,63 @@
+import ReactElement = __React.ReactElement;
+'use strict';
+
+import React = require('react');
+import Bootstrap = require('react-bootstrap');
+import _ = require('underscore');
+import {IOrder} from "../../../../backend/src/order";
+
+
+export interface IFinalConfirmationProps {
+  order: IOrder;
+  paymentBegun: boolean;
+
+  onProceedToPayment: Function;
+}
+
+export default class FinalConfirmation extends React.Component<IFinalConfirmationProps, any> {
+
+  render() {
+    if(!this.props.order) {
+      return (<div />);
+    }
+    var numTickets = this.props.order.tickets.length;
+    var finalPrice = this.props.order.order_price;
```

Comment:
even though order_price seems to come correctly from backend, frontend loses it somehow (checking React console for FinalConfirmation, it shows that this.props.order.order_price = null)